app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -255,10 +256,13 @@
 
     private static String statusIOB = "";
     private static String statusBWP = "";
+    private NotificationChannels notifChannels;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         mActivity = this;
+        // Builds notification channels if supported
```

Comment:
What happens if a notification is raised before `NotificationChannels` is instantiated? It is quite possible for `Home.onCreate()` to never be called. For example if the user reboots their phone, doesn't open the app and then receives a notification. If this is always needed then it should be appended to the initialization in the `xdrip` Application class I think.

Reply:
Was wondering about this case; I was assuming that Home would be loaded early enough that this wouldn't matter, but that makes total sense. You're talking about onCreate() in xdrip.java, correct?